packages/sites/src/drafts/save-published-status.ts

```diff
@@ -2,6 +2,7 @@ import { IModel, IHubRequestOptions } from "@esri/hub-common";
 import { updateSite } from "../update-site";
 import { updatePage, isPage } from "../pages";
 import { isSite } from "../is-site";
+import { IUpdateItemResponse } from "@esri/arcgis-rest-portal";
```

Comment:
if this interface is new in 2.17, we need to bump `@esri/arcgis-rest-portal` in the sites' `peerDependencies` as well as the `devDependencies`

Reply:
sorry, I didn't mean to block this - I meant to add that as a comment, not request for change.

And yes, you were on exactly the right track.

I'm going to approve this PR, it's good as is as long as `IUpdateItemResponse` was not added in 2.17